plugins/fields.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: fields
+spec:
+  platforms:
+  - uri: https://github.com/rewanth1997/kubectl-fields/releases/download/v1.0.1/kubectl-fields-Darwin-x86_64.tar.gz
+    sha256: 9fe7b746fa74ba3deedabf77ecb35e7e24b7fc0410dd96219dff4ece02543fdf
+    bin: kubectl-fields
+    files:
+    - from: "*"
+      to: "."
+    selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+  - uri: https://github.com/rewanth1997/kubectl-fields/releases/download/v1.0.1/kubectl-fields-Linux-x86_64.tar.gz
+    sha256: 21ee6334120bae36ab4a4b23fed731423d81b21058a6df33e92b38110e0fbb2b
+    bin: kubectl-fields
+    files:
+    - from: "*"
+      to: "."
+    selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+  - uri: https://github.com/rewanth1997/kubectl-fields/releases/download/v1.0.1/kubectl-fields-Windows-x86_64.zip
+    sha256: 1eea1afb8b623934a240d9cc31923e9ab036524ae3d8a8349b50daced7e99cd7
+    bin: kubectl-fields.exe
+    files:
+    - from: "*"
+      to: "."
+    selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+  version: v1.0.1
+  homepage: https://github.com/rewanth1997/kubectl-fields
+  caveats: |
+    Usage:
+      $ kubectl fields <RESOURCE> [options] <PATTERN> [<PATTERN 2> ...]
+
+    Options:
+      -h, --help          help for kubectl-fields
+      -i, --ignore-case   Ignore case distinction
+      --stdin         Expects input via pipes
+
+    Read the documentation:
+      https://github.com/rewanth1997/kubectl-fields/blob/master/README.md
+  shortDescription: Kubectl resources hierarchy parser
+  description: |
+    Kubectl resources hierarchy parser.
+
+    More on https://github.com/rewanth1997/kubectl-fields
+
+    Examples:
+
+    $ kubectl fields po.spec capa
+    containers.securityContext.capabilities
```

Comment:
Even though the plugin output won't work like this, you could think about highlighting the matching portion of the string in bold using shell escape sequences. This will help to understand how the plugin works.

Reply:
I added colors to this using https://github.com/fatih/color
Please review it

Reply:
Regarding colors on windows terminals, this might be of interest to you: https://github.com/Azure/draft/pull/928

Reply:
Perfect. Done.